packages/cms/custom_migrations/20220825A-migrate-price-currencies.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable */
+
+module.exports = {
+  async up(knex) {
+    // add default USD, ETH for currencies
+    await knex('currencies').insert({ currency: 'USD' });
+    await knex('currencies').insert({ currency: 'ETH' });
+
+    const priceCurrencies = await knex('price_currencies');
+    const products = await knex('products').whereNotNull('price');
+    const currencies = await knex('currencies');
+
+    const getThingByKey = (things, key, value) => {
```

Comment:
Could also use Lodash keyBy: https://lodash.com/docs/#keyBy

Reply:
is lodash available inside the directus env?

Reply:
ah sick